app/assets/images/index.ts

```diff
@@ -0,0 +1,11 @@
+import ChevronDownIcon from "./chevron-down.svg"
+import ClipboardIcon from "./clipboard.svg"
+import HelpIcon from "./help.svg"
+import StructureOfJwt from "./structure-of-jwt.png"
```

Comment:
This asset is really large, is the image size the same as the usage size? If yes that whats the display size? Either way ~400kb is massive and we should find a way to reduce it